samtranslator/model/sam_resources.py

```diff
@@ -456,8 +459,8 @@ def _make_gen_condition_name(self, name: str, hash_input: str) -> str:
         # Make sure the property name is not over 255 characters (CFN limit)
```

Comment:
Nit: Could reword as "is not over the CFN limit (currently 255)". I doubt this limit will ever change, so this is probably not a big deal.

Reply:
updated